pkg/ui/src/util/docs.ts

```diff
@@ -12,14 +12,20 @@
 
 import { getDataFromServer } from "src/util/dataFromServer";
 
-const version = getDataFromServer().Version || "stable";
+const stable = "stable";
+const version = getDataFromServer().Version || stable;
```

Comment:
Welp, the version will still be in dataFromServer, which is plopped directly into the HTML:

```html
<!DOCTYPE html>
<html>
	<head>
		<title>Cockroach Console</title>
		<meta charset="UTF-8">
		<link href="favicon.ico" rel="shortcut icon">
	</head>
	<body>
		<div id="react-layout"></div>

		<script>
			window.dataFromServer = {"ExperimentalUseLogin":true,"LoginEnabled":true,"LoggedInUser":null,"Tag":"v19.2.0-alpha.20190606-250-g8dc92a6","Version":"v19.2","NodeID":"1"};
		</script>

		<script src="protos.dll.js" type="text/javascript"></script>
		<script src="vendor.dll.js" type="text/javascript"></script>
		<script src="bundle.js" type="text/javascript"></script>
	</body>
</html>
```

So AFAIK it's even easier to scrape it from there than from the actual rendered page, since to render the page the bot would need JS. So, idk.

Reply:
@vilterp I'm looking at `util/dataFromServer.ts`:
```
export interface DataFromServer {
  ExperimentalUseLogin: boolean;
  LoginEnabled: boolean;
  LoggedInUser: string;
  Tag: string;
  Version: string;
  NodeID: string;
}

// Tell TypeScript about `window.dataFromServer`, which is set in a script
// tag in index.html, the contents of which are generated in a Go template
// server-side.
declare global {
  interface Window {
    dataFromServer: DataFromServer;
  }
}

export function getDataFromServer(): DataFromServer {
  return window.dataFromServer || {} as DataFromServer;
}
```
One one hand this seems like we can't get around this issue without altering the data we're pulling in from the server.  On the other it seems like the only place in the codebase that references the `Version` is in this file.  I don't know enough about customer use cases to know if we can safely remove version data (cc: @piyush-singh)...does that sound like too consequential a change to quickly include?  If so, it seems like a more sophisticated system of toggling public version data would require a bit more thinking and development.

Reply:
Yeah, would have to make the change in #38337 (which kinda breaks docs links), plus scrubbing other endpoints
- prometheus endpoint has version info I think
- `/health` has version info